packages/core/src/internal-components/overlay/overlay.stories.ts

```diff
@@ -87,6 +90,9 @@ export const interactive = () => {
       myOverlay.addEventListener('closeChange', () => {
         myOverlay.setAttribute('hidden', 'true');
```

Comment:
for consistency, should we set attribute value to empty quotes here as well?

Reply:
The stories below all follow this pattern so I won't add additional comments. I figured they are all going to stay the same or be updated together. :)

Reply:
maybe? i'm not sure. i'd have to test it against the property-driven animation to make sure that worked as expected. i think it would but 🤷‍♂️

Reply:
it works! change made. thanks!